orange-datamodel/src/test/resources/real.orange.json

```diff
@@ -163,7 +164,8 @@
           "effects": [
             "MISSENSE"
           ],
-          "codingEffect": "MISSENSE"
+          "codingEffect": "MISSENSE",
+          "reported": true
         },
```

Comment:
This file was created from COLO using `ReportLimiter`. This logic doesn't guarantee a self-consistent dataset, but think that is not necessary either. That being said, the name could be improved, or we could even  decide to merge with minimally populated or delete it, it doesn't really add a lot. At this point, the minimally populated is manually managed, while the real is generated from actual data. 

I don't have a strong opinion on what should change (if anything), but would be good to generate the `real.orange.json` from the actual Orange Test Application.

Reply:
Oh, I see. I just ran this test application, presumably all the (unrelated to this PR) differences I see in the output are to be expected, e.g. platinum version 5.34 etc, I think all from the test files that must have been updated since the previous time real.orange.json was generated.

If that sounds right, I can check this in (after running jq to for layout). One odd thing is that the current version has unicode characters in the variants for the > char, wonder how that happened 🤔. The version I generated seems ok though.

Reply:
Yes, would be good to refresh this file with all changes included.